app.py

```diff
@@ -0,0 +1,30 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
+
+BASE_URL = "https://api.nytimes.com/svc/search/v2/articlesearch.json"
+
+params = {
+    "q": "braves",  # Query keywords
+    "api-key": os.getenv(MY_API_SECRET_KEY),
```

Comment:
Why is os.getenv being used when the api key is in the app.py itself? getenv is used for getting from a .env file.